src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -113,6 +114,23 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
         Power = PWR_25mW;
         break;
     }
+#elif defined(TARGET_RX_BETAFPV_2400_V1)
+    switch (Power)
+    {
+    case PWR_10mW:
+        Radio.SetOutputPower(-10);
+        break;
+    case PWR_50mW:
+        Radio.SetOutputPower(-3);
+        break;
+    case PWR_100mW:
+        Radio.SetOutputPower(1);
+        break;
+    default:
+        Radio.SetOutputPower(1);
+        Power = PWR_100mW;
```

Comment:
The `case PWR_100mW:` can fall through to the `default`, like:
```
case PWR_100mW:
default:
        Radio.SetOutputPower(1);
        Power = PWR_100mW;
```
And you can then get rid of the 100mW case.